index.js

```diff
@@ -92,16 +93,39 @@ function render(data, meta, res) {
   });
 
   if (_.get(meta, 'locals.query.config', false)) {
-    _.assign(output, getSiteConfig(meta.locals.site));
-    output.siteSlug = meta.locals.site.slug;
+    const requestedSite = _.get(meta, 'locals.query.replacement', ''),
+      siteMetadata = meta.locals.site,
+      siteData = requestedSite ? replaceSiteDir(siteMetadata, requestedSite) : siteMetadata;
+
+    _.assign(output, getSiteConfig(siteData));
+    output.siteSlug = siteMetadata.slug;
   }
 
   res.json(output);
 }
 
+/**
+ * Replaces the site dir to use the desired site path
+ * @param {Object} siteMetadata
+ * @param {string} siteMetadata.dir
+ * @param {string} siteMetada.slug
+ * @param {string} replacement
+ * @returns {Object}
+ */
+function replaceSiteDir({ dir = '', slug = '' }, replacement) {
+  if (!replacement) return { dir };
+
+  const SITE_DIR_REGEX = new RegExp(`(\/${slug})$`);
```

Comment:
I think this should be replaced with a method that mimics [getComponentPath](https://github.com/clay/amphora-fs/blob/master/lib/getComponentPath/index.js) called `getSitePath` which is a method which given a site slug will regenerate the path to the that site's config directory

For example:

```js
getSitePath('hello') === `${process.cwd()}/sites/hello`
```

Reply:
For now we can just make the method in this repository, at a later time we can evaluate if it should be merged up into `amphora-fs`